cmd/buildkitd/main.go

```diff
@@ -256,6 +269,73 @@ func main() {
 		}
 
 		opts := []grpc.ServerOption{grpc.UnaryInterceptor(unary), grpc.StreamInterceptor(stream)}
+
+		kaspMintimeDuration := defaultKeepalivePolicyMinTime
+		if kaspMintimeEnv := os.Getenv("OKTETO_KEEPALIVE_SERVER_POLICY_MINTIME_MS"); kaspMintimeEnv != "" {
+			kaspMintimeDuration, _ = time.ParseDuration(fmt.Sprintf("%sms", kaspMintimeEnv))
```

Comment:
Be careful, you are not catching the error that you check below

Reply:
In general, the same review as the other file:
- Handle the errors
- Log when envvar cannot be parsed